amical/mf_pipeline/bispect.py

```diff
@@ -997,7 +1010,6 @@ def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix):
         # https://github.com/astropy/astropy/issues/11866
         # Resolved upstream with
         # https://github.com/astropy/astropy/pull/11923
-        hdr_commentary_keys = fits.Card._commentary_keywords
```

Comment:
I'm not sure why this line was removed. @vandalt could you explain your reasoning ?

Reply:
It was moved out of the `if` statement to issue the warning only when there is actually a "commentary key" in the header (i.e. when something will indeed be removed form the header). I can add a comment before the `if` to clarify that.

Reply:
I added the comment, marking as resolved.